options/locale/locale_en-US.ini

```
@@ -2308,17 +2308,17 @@ settings.matrix.room_id = Room ID
 settings.matrix.message_type = Message Type
 settings.archive.button = Archive Repo
 settings.archive.header = Archive This Repo
-settings.archive.text = Archiving the repo will make it entirely read-only. It is hidden from the dashboard, cannot be committed to and no issues or pull-requests can be created.
+settings.archive.text = Archiving the repo will make it entirely read-only. It will be hidden from the dashboard. Nobody (not even you!) will be able to make new commits, issues and pull requests.
 settings.archive.success = The repo was successfully archived.
 settings.archive.error = An error occurred while trying to archive the repo. See the log for more details.
 settings.archive.error_ismirror = You cannot archive a mirrored repo.
 settings.archive.branchsettings_unavailable = Branch settings are not available if the repo is archived.
 settings.archive.tagsettings_unavailable = Tag settings are not available if the repo is archived.
-settings.unarchive.button = Un-Archive Repo
-settings.unarchive.header = Un-Archive This Repo
-settings.unarchive.text = Un-Archiving the repo will restore its ability to receive commits and pushes, as well as new issues and pull-requests.
-settings.unarchive.success = The repo was successfully un-archived.
-settings.unarchive.error = An error occurred while trying to un-archive the repo. See the log for more details.
+settings.unarchive.button = Unarchive Repo
+settings.unarchive.header = Unarchive This Repo
```

Why titlecase all of a sudden?
```suggestion
settings.unarchive.button = Unarchive repo
settings.unarchive.header = Unarchive this repo
```